tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -37,6 +38,41 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     Then "Anonymous" should not be able to edit content of following resource
       | resource         | type         | content                     |
       | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
+
+    When "Alice" shares the following resource using the sidebar panel
+      | resource         | recipient | type | role     | resourceType |
+      | OpenDocument.odt | Brian     | user | Can view | file         |
```

Comment:
@nirajacharya2 Should the scenario flow be exact like it is in https://github.com/owncloud/web/issues/10750 ?

Reply:
@anon-pradip Please, confirm that this behavior is checked

Reply:
Regarding https://github.com/owncloud/ocis/issues/8060
After the 4th step i.e open the file and don't close it as `marie`, the permission can not be changed for:
5th step i.e change the permission to view as `admin`.
Since the file is opened in `marie`'s editor, we can not change the permission because the file is locked with hover warning `Resource is temporarily locked, unable to manage share` in the permission changing dropdown.